src/PostgREST/Middleware.hs

```diff
@@ -48,6 +55,13 @@ runWithClaims conf eClaims app req =
       ]
       (toS $ "{\"message\":\""<>message<>"\"}")
 
+envVarToSql :: Text -> [(Text, Text)] -> [ByteString]
+envVarToSql prefix vars = varStmts
```

Comment:
This is minor, but you could inline `varStmts` as `map setVar vars` here. The extra name doesn't get us too much.